tests/components/zwave_mqtt/conftest.py

```diff
@@ -0,0 +1,42 @@
+"""Helpers for tests."""
+import json
+
+from asynctest import patch
+import pytest
+
+from .common import MQTTMessage
+
+from tests.common import load_fixture
+
+
+@pytest.fixture(name="generic_data")
```

Comment:
Can we change the fixture scope so it will only run once per test run ?

Reply:
No, we need the loop fixture which is test scope only. I already tried.

Reply:
That's because you made it an async fixture. Make it a sync fixture and call load_fixture direectly.

Reply:
Of course!

Reply:
Commit done.